dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -66,6 +68,9 @@ def update
     section = Section.find(params[:id])
     authorize! :manage, section
 
+    # Can't update participant_type once it is set
+    return head :forbidden if !params[:participant_type].nil_or_empty? && params[:participant_type] != section.participant_type
```

Comment:
this is ok to include, but I would just note that in my opinion this is probably going above and beyond in terms of trying to prevent the client for misusing the API. because the update API is ignoring this param, I think you could also remove this line (and corresponding test).